rsmtool/rsmtool.py

```diff
@@ -109,6 +111,15 @@ def run_experiment(config_file_or_obj_or_dict, output_dir, overwrite_output=Fals
     logger.info("Saving configuration file.")
     configuration.save(output_dir)
 
+    # If wandb logging is enabled, start a wandb run and log configuration
+    use_wandb = configuration.get("use_wandb", False)
+    wandb_run = None
+    if use_wandb:
+        wandb_project = configuration.get("wandb_project")
```

Comment:
I think these `.get`s should just be direct accesses. We probably don't want them ending up as `None`s. But also the validation checks for the presence of both fields, so we currently wouldn't get here without having them both be present.